panel/pane/base.py

```diff
@@ -441,14 +450,14 @@ def _cleanup(self, root=None):
         self._inner_layout._cleanup(root)
         super()._cleanup(root)
 
-    def select(self, selector=None):
+    def select(self, selector: Union[type, Callable, None]=None) -> List[Viewable]:
         """
         Iterates over the Viewable and any potential children in the
         applying the Selector.
 
         Arguments
         ---------
-        selector: type or callable or None
+        selector: (type | callable | None)
```

Comment:
For consistency with the way you can construct unions of types from Python 3.10 I changed the docstring.